test/unit/serialize.js

```diff
@@ -335,16 +335,18 @@ describe('serialize( obj )', function () {
             var regexKey = /.*/;
             var m = new Map([
                 ['a', 123],
-                [regexKey, 456]
+                [regexKey, 456],
+                [Infinity,322]
             ]);
-            expect(serialize(m)).to.be.a('string').equal('new Map([["a",123],[new RegExp(".*", ""),456]])');
-            expect(serialize({t: [m]})).to.be.a('string').equal('{"t":[new Map([["a",123],[new RegExp(".*", ""),456]])]}');
+            expect(serialize(m)).to.be.a('string').equal('new Map([["a",123],[new RegExp(".*", ""),456],[Infinity,322]])');
+            expect(serialize({t: [m]})).to.be.a('string').equal('{"t":[new Map([["a",123],[new RegExp(".*", ""),456],[Infinity,322]])]}');
         });
 
         it('should deserialize a map', function () {
             var m = eval(serialize(new Map([
                 ['a', 123],
-                [null, 456]
+                [null, 456],
+                [Infinity,322]
```

Comment:
I think it should be `789` to match the above regularity.

Reply:
I failed to notice the sequential suit. Here you are: 6ac6afe